token/js/src/instructions/initializeMultisig.ts

```diff
@@ -43,7 +44,11 @@ export function createInitializeMultisigInstruction(
         { pubkey: SYSVAR_RENT_PUBKEY, isSigner: false, isWritable: false },
     ];
     for (const signer of signers) {
-        keys.push({ pubkey: signer, isSigner: false, isWritable: false });
+        keys.push({
```

Comment:
Using `addSigners` here would result in a different keys array, which makes me think these keys aren't correct...

Reply:
Correct, you can't make the change here